components/ADempiere/FileRender/index.vue

```diff
@@ -60,6 +60,10 @@ export default defineComponent({
   },
 
   setup(props) {
+    console.log({
```

Comment:
@elsiosanchez please remove this console

Reply:
Ready. console removed